scripts-available/CDB_Overviews.sql

```diff
@@ -531,6 +551,54 @@ AS $$
   );
 $$ LANGUAGE SQL STABLE;
 
+CREATE OR REPLACE FUNCTION _cdb_categorical_column(reloid REGCLASS, col_name TEXT)
+RETURNS BOOLEAN
+AS $$
+DECLARE
+    schema_name TEXT;
+    table_name TEXT;
+    available BOOLEAN;
+    categorical BOOLEAN;
+BEGIN
+    SELECT * FROM _cdb_split_table_name(reloid) INTO schema_name, table_name;
+    SELECT n_distinct IS NOT NULL
```

Comment:
If stats do no exist n_distinct will be NULL. Checking booleans is tricky with NULLs involved...